packages/curve-ui-kit/src/shared/ui/ActionInfo.tsx

```diff
@@ -98,18 +104,18 @@ const ActionInfo = ({
   loading = false,
   error = false,
   testId = 'action-info',
-  sx,
+  ...styleProps // sx, className
 }: ActionInfoProps) => {
   const [isSnackbarOpen, openSnackbar, closeSnackbar] = useSwitch(false)
-  const copyValue = (givenCopyValue ?? value).trim()
+  const copyValue = (givenCopyValue ?? (typeof value === 'string' ? value : '')).trim()
```

Comment:
Should we really fail silently if value is a `ReactNode`? Perhaps copying should be disabled altogether if it's not a string?

Debatable because you've already used `copy={typeof children === 'string'}` in the `DetailInfo` component, but maybe it's better to put that guard internally into the component so it can't be forgotten.